### ISSUE TITLE
fog shader not working as expected [bug]

### ISSUE THREAD

**Issue by LocalGod79** [endless-sky/endless-sky#1381]
on my system, the galactic map is completely obscured, no matter how many systems i explore.

i'm not really sure how much this feature makes sense (and i've always found adds to the intimidation factor of a new game) - we already know what our galaxy looks like with the level of detail that is displayed in the graphic; why is the Republic censoring information from ship navcomputers that could theoretically be collected from even ground based telescopes and a wristwatch calculator?

**Comment by KiLEdEnNis** [comment]
it adds a certain feel to the game though, because usually if you explore a lot the edges seem empty, but maybe make it so that human space isn't obscured

**Comment by LocalGod79** [comment]
i get that it adds a "certain feel".  i'm just not sure that it's congruous.

let me try different wording.

just because you've never been to England (or pick a country you've not been to), doesn't mean you've not seen it on a map or globe and know generally where it is.  it does mean that if you sailed to London (or your chosen country's primary port), that you'd "know the way to get there" - and you'd still know what the globe looked like, even if you didn't know where all the cities were and "how to get to" each one.

in short, it seems ridiculous that a spacefaring civilization would have starship captains leaving their planets without knowing what the galaxy looked like (or at least their controlled region), or what their civilization had named given regions within their control.

that it's also buggy only adds to my dislike of this feature (and perhaps my critique is unfair, since i can't see how much background image is "being revealed" based on my location).

**Comment by endless-sky** [comment]
Well, if it were just a matter of realism you should also start out with a
map of all of human space. But discovering the map little by little gives
you a sense of exploration, and I'm hoping the "fog of war" will have the
same effect. (But, if I can't track down your bug, it'll have to be turned
off by default in the next release anyway.)

A new player starts off with just enough map visible to see the labels for
"South" and "Earth," which might actually make exploration a bit *less*
intimidating at first.

@LocalGod79, are you running with either a high-DPI monitor or the zoom
level in the preferences set to something other than 100%? I didn't test
that due to my small monitor, so that's a likely culprit. But, I'll also
try testing on OS X when I get the chance to see if the bug is OS-specific.

**Comment by LocalGod79** [comment]
using a 1080p monitor (two actually, but they're both 1080p), normal resolution sprites and zoom.

**Comment by KiLEdEnNis** [comment]
yes but you can't exactly draw visible outlines around human space for that matter, that'd look weird and half-retarded

**Comment by endless-sky** [comment]
@LocalGod79, check if the latest code fixes this. It turns out that on OS X OpenGL can't work with textures that are alpha-only, so I switched to a texture with only a red channel.

**Comment by LocalGod79** [comment]
this is now working as i believe you intended, and i'll follow up with feedback: the radius appears large enough to not be too intimidating (at least there are some labels visible)
